src/service/action-impl.js

```diff
@@ -152,11 +158,33 @@ export class ActionService {
           this.trigger(dev().assertElement(event.target), 'tap', event);
         }
       });
-    } else if (name == 'submit' || name == 'change') {
+    } else if (name == 'submit') {
       this.root_.addEventListener(name, event => {
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'change') {
+      this.root_.addEventListener(name, event => {
+        const target = event.target;
+        const changeEvent = this.getChangeDetails_(event);
+        this.trigger(dev().assertElement(target), name, changeEvent);
+      });
+    }
+  }
+
+  getChangeDetails_(event) {
+    const details = {};
+    const target = event.target;
+    if (event.target.tagName.toLowerCase() === 'input') {
+      const inputType = target.getAttribute('type');
+      const fieldsToInclude = WHITELISTED_INPUT_DATA_[inputType];
+      // TODO(kmh287): Cast numeric values to numbers?
```

Comment:
TBH it was initially a question. I'm not sure what's the right thing to do here. Does anyone have any other opinion? There's a bit of a conflict here before us preferring to operate with the actual types and (looks like) `input/range` values appear to be strings in Chrome? Am I seeing this right? This is actually pretty strange and might be worth confirming. If I read the spec right, they all should be fp numbers already: https://www.w3.org/TR/html5/forms.html#range-state-(type=range)

Might be worth filing a bug against Chrome and proceed with what we'd prefer downstream. One option is to add types in the `WHITELISTED_INPUT_DATA_` config. Alternatively, we can sniff for typed values. E.g. `!isNaN(input.valueAsNumber)` or such.